src/pages/Movie/TopInfo/ToggleToFavorite/__tests__/containet.test.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
```

Comment:
Please, rename this file from `containet.test.js` to `container.test.js`

Reply:
Done